### ISSUE TITLE
[a11y] 9.2.4.7 - Current focus position visible

### ISSUE THREAD

**Issue by tbsbdr** [owncloud/web#5393]
- Context: The Web UI should be accessible according to the WCAG2.1 /BITV 2.0 guidelines. The number in the title relates to the test-steps: https://www.bitvtest.de/bitv_test/das_testverfahren_im_detail/pruefschritte.html
- "Status" relates to the evaluation scheme https://www.bitvtest.de/bitv_test/das_testverfahren_im_detail/verfahren.html#c1262 
- "a11y" is short for accessibility
- ----------------
## General notes
The highlighting of the focus (current focus position) via keyboard works with Chrome/EDGE. When using Firefox and Safari the focus borders are sometimes cut off. With Firefox the presentation of the focus border varies which leads to irritation for the users.

**Comment by stevefrenzel** [comment]
@tbsbdr @kulmann Confluence page says the following: 

"Debatable if it needs fixing for the re-audit (partly fulfilled and browser-specific)"

Does it still need to be fixed?

If so: Cut off borders in Safari are caused by "overlapping" padding and or margin of other components. Unfortunately this is a Safari exclusive problem, so it could cause layout issues in other browsers. 😕

Consistent border styles could be achieved by removing browser specific design and implementing a custom one.

**Comment by pascalwengerter** [comment]
Superseded by the more recent #10735
